tests/frontend/test_kibana_app/cypress/integration/step-definitions/filters/filters-are-displayed-in-agents-modules.and.js

```diff
@@ -0,0 +1,9 @@
+import { And } from 'cypress-cucumber-preprocessor/steps';
+import { elementIsVisible, getElement} from '../../utils/driver';
+import {
+    stablishedFilter
+  } from '../../pageobjects/filters/filters.page';
+
+Then('The user check filter label is added', () => {
```

Comment:
I know this will be changed when you change the gherkin files but remember 3rd person verbs end with 's'/'es' in the present tense